synapse/server_notices/server_notices_manager.py

```diff
@@ -63,7 +63,7 @@ async def send_notice(
             is_state_event(bool): Is the event a state event
 
         Returns:
-            Deferred[FrozenEvent]
+            FrozenEvent
```

Comment:
Why not give this function the type signature treatment? :slightly_smiling_face:

Reply:
Looks like I missed this file!

Reply:
Fixed this! Thanks for noticing. 👍